docs/master/subscriptions/client-implementations.md

```diff
@@ -92,11 +92,8 @@ class PusherLink extends ApolloLink {
 
   getChannelFromResponse(response, operation) {
     return !!response.extensions &&
```

Comment:
How about `response?.extensions?.lighthouse_subscriptions.channel`?

Reply:
Damn that was quick! 👍 
If the `lighthouse_subscriptions` is defined, `channel` is always defined. If no channel, the value will be `null` :-)

Reply:
Sunday evening, typical working hours for Open Sourcerers 🧙